Updated _misplaced_members for PySide6

johnsonthomassonttf opened the pull request:
Long time listener, first time caller from [The Third Floor](https://thethirdfloorinc.com/)

I located these 4 members in PySide6 and based this modification on similar commits. Please let me know if there's a better way to do this. We're actively testing in PySide6 and PySide2, so I may discover more misplaced members.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/mottosso/Qt.py?pullRequest=405) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/mottosso/Qt.py?pullRequest=405) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/mottosso/Qt.py?pullRequest=405) it.</sub>

mottosso commented:
Thanks! Yes, this is the way, except we also need them for PyQt4 and PyQt5. It's hard to miss right now.. And tests won't pick this up.. But if they aren't there, then they simply would not get added to Qt.py when using those.

Could you add these to PyQt4 and 5 as well?

johnsonthomassonttf commented:
Added. Also added to PySide. I couldn't find API docs for PyQt4, but I believe it should be a match for PySide 1, so I referenced those API docs. Interesting that they flip-flopped on the location of these members.

mottosso commented:
Looks good, thanks @johnsonthomassonttf!
